common/views/themes/config.ts

```diff
@@ -257,4 +321,37 @@ export const themeValues = {
 };
 
 export type Breakpoint = keyof typeof sizes;
-export type PaletteColor = keyof typeof colors;
+
+// TODO: Is there an automated way to do this?
+export type newPaletteColor =
```

Comment:
yeah this makes me unhappy, I'll have to investigate how this could be done otherwise.